src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -257,6 +259,30 @@ public final String getMetaFullName() {
         return getMetaName() + ".name";
     }
 
+    public <T> void addNotifiedInput(T input) {
+        if (input instanceof IItemHandlerModifiable) {
+            if (!notifiedItemInputList.contains(input)) {
+                this.notifiedItemInputList.add((IItemHandlerModifiable) input);
+            }
+        } else if (input instanceof FluidTank) {
+            if (!notifiedFluidInputList.contains(input)) {
+                this.notifiedFluidInputList.add((FluidTank) input);
+            }
+        }
+    }
+
+    public <T> void addNotifiedOutput(T output) {
```

Comment:
The parameterized type <T> on this and the addNotifiedInput() does nothing?
It could just be replaced with Object.
You are using instanceof on the parameter anyway.